test/runtests.jl

```diff
@@ -205,6 +205,33 @@ function test_l1b_bool_mask_with_fake_data()
 
 end
 
+function test_mask_dilate()
+
+  mask = zeros(Int64, 5, 5)
+  mask[3, 3] = 1
+  kern = zeros(Int64, 3, 3) + 1
+
+  include(joinpath(Pkg.dir("KillerAsteroids"), "src", "mask_dilate.jl"))
```

Comment:
Maybe this line should be in `src/KillerAsteroids.jl` with all the other `include`  statements?